eclide/mainfrm.cpp

```diff
@@ -2188,16 +2351,6 @@ bool CMainFrame::UIUpdateMenuItems()
     return mdiActive; //
 }
 
-BOOL CMainFrame::DoFileOpen(const CString & sPathName, int row, int col, int len)
-{
-    boost::filesystem::path path = stringToPath(static_cast<const TCHAR *>(sPathName));
-    std::_tstring filename = pathToWString(path.leaf());
-    CComPtr<IRepository> rep = AttachRepository();
-    AttrInfo attrInfo;
-    attrInfo.AttributeType = ExtensionWithoutDot(filename);
-    return OpenFileBuilderMDI(this, attrInfo, sPathName, rep->CreateIWorkspaceItem(WORKSPACE_ITEM_BUILDER, filename, static_cast<const TCHAR *>(sPathName)), false, row, col, len);
-}
```

Comment:
Is this not needed for opening an editor at a specific location (syntax errors)?